prov/efa/src/efa_av.c

```diff
@@ -527,26 +558,61 @@ static int efa_av_remove(struct fid_av *av_fid, fi_addr_t *fi_addr,
 	int ret = 0;
 	size_t i;
 	struct efa_av *av;
+	struct util_av_entry *util_av_entry;
 	struct efa_av_entry *av_entry;
-	struct efa_ep_addr addr;
+	struct rxr_peer *peer;
+	struct rxr_ep *rxr_ep;
+	struct dlist_entry *ep_list_entry;
+	struct util_ep *util_ep;
 
 	av = container_of(av_fid, struct efa_av, util_av.av_fid);
 	if (av->ep_type == FI_EP_RDM) {
```

Comment:
It might be worth negating this block since the DGRAM case is short, that way you can reduce the amount of indentation here.